docs/contributing.md

```diff
@@ -61,6 +61,15 @@ Not everything is enforced or validated by the schema. A few things to pay atten
 - Nesting of feature identifiers should make sense.
 - Notes use correct grammar and spelling. They should be complete sentences ending with a period.
 
+### Old pull requests
+
+We may close pull requests that have been inactive for 30 or more days.  The procedure is as follows:
```

Comment:
Update "inactive for 30 or more days." to "inactive for an extended period of time." We mention the 30 day estimate in the last sentence; no point in saying it twice.